### PR TITLE
feat: add reply object and reply address object

### PR DESCRIPTION
**Pull request by magicmatatjahu**
**Description**

Add `reply` object and `reply address` object.

**Related issue(s)**
Part of https://github.com/asyncapi/spec/pull/847

### CODE REVIEW

File: definitions/3.0.0/components.json [hunk 1]
```diff
@@ -68,10 +68,34 @@
         "$ref": "http://asyncapi.com/definitions/3.0.0/messageTrait.json"
       }
     },
-    "serverBindings": {
```
Comment (fmvilas) [inline definitions/3.0.0/components.json:71]: Why are you removing the `serverBindings`? 🤔

Comment (magicmatatjahu) [reply definitions/3.0.0/components.json:71]: LOL, good catch! 👀

**Review by char0n — APPROVED** [review]
LGTM.

File: definitions/3.0.0/operationReply.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+          "$ref": "http://asyncapi.com/definitions/3.0.0/Reference.json"
+        },
+        {
+          "$ref": "http://asyncapi.com/definitions/3.0.0/operationReplyAddress.json"
+        }
+      ]
+    },
+    "channel": {
+      "$ref": "http://asyncapi.com/definitions/3.0.0/Reference.json"
+    }
```
Comment (smoya) [inline definitions/3.0.0/operationReply.json:22]: @magicmatatjahu `messages` field is missing. See https://github.com/asyncapi/spec/blob/next-major-spec/spec/asyncapi.md#operation-reply-object

File: definitions/3.0.0/operationReply.json [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+      "oneOf": [
+        {
+          "$ref": "http://asyncapi.com/definitions/3.0.0/Reference.json"
+        },
+        {
+          "$ref": "http://asyncapi.com/definitions/3.0.0/operationReplyAddress.json"
+        }
+      ]
+    },
+    "channel": {
```
Comment (smoya) [inline definitions/3.0.0/operationReply.json:20]: @magicmatatjahu messages field is missing. See https://github.com/asyncapi/spec/blob/next-major-spec/spec/asyncapi.md#operation-reply-object

**Review by smoya — CHANGES_REQUESTED** [review]

File: definitions/3.0.0/operationReply.json (hunk 3 above)
Comment (magicmatatjahu) [reply definitions/3.0.0/operationReply.json:20]: Field was created before my PR. I will add that.

Comment (magicmatatjahu) [reply definitions/3.0.0/operationReply.json:20]: @smoya Added :) Can you accept?

**Review by smoya — APPROVED** [review]
LGTM! 🚀🌔

**Pull request merged**